docs/site/content/docs/advanced/cloudrootfs.md

```diff
@@ -9,7 +9,111 @@ sort_by = "weight"
 template = "docs/page.html"
 
 [extra]
-lead = "Answers to frequently asked questions."
+lead = "CloudRootfs will package all the dependencies refers to the kubernetes cluster requirements."
 toc = true
 top = false
-+++
\ No newline at end of file
++++
+
+# What is CloudRootfs
+
+All the files witch run a kubernetes cluster needs.
+
+Contains:
+
+* Bin files, like docker containerd crictl kubeadm kubectl...
+* Config files, like kubelet systemd config, docker systemd config, docker daemon.json...
+* Registry docker image
+* Some Metadata, like Kuberentes version.
+* registry files, contains all the docker image, like kubernetes core components docker images...
+* scripts, some shell script using to install docker and kubelet... sealer will call init.sh and clean.sh.
+* Other static files
```

Comment:
* Scripts, some shell script using to install docker and kubelet... sealer will call init.sh and clean.sh.
* Other static files.